ios/RTCPjSip/PjSipAccount.m

```diff
@@ -116,6 +116,36 @@ - (id)initWithConfig:(NSDictionary *)config {
                 }
             }
         }
+        // TURN settings
+        NSDictionary *turnConfiguration = config[@"turn"];
+        if (turnConfiguration) {
+            NSString *turnServer = turnConfiguration[@"server"];
+            NSString *transportType = turnConfiguration[@"tp_type"];
+            NSString *realm = turnConfiguration[@"realm"];
+            NSString *username = turnConfiguration[@"username"];
+            NSString *passwordDataType = turnConfiguration[@"password_data_type"];
```

Comment:
It would be great if we will have the same structures as in pjsip, I mean

```
const turnConfiguration = {
  server: "XXX",
  connectionType: "UDP", // TCP , TLS
  authentication: {
    realm: "XXX",
    username: "XXX",
    dataType: "PLAIN", // "HASHED"
    data: "XXX"
  }